.jenkins/pytorch/win-test.sh

```diff
@@ -55,7 +55,7 @@ fi
 
 run_tests() {
     # Run nvidia-smi if available
-    for path in  /c/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi.exe /c/Windows/System32/nvidia-smi.exe; do
+    for path in '/c/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi.exe' /c/Windows/System32/nvidia-smi.exe; do
```

Comment:
What is the reason for this change?

Reply:
`/c/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi.exe` without the quotes represents three arguments in the for loop:

* `/c/Program`
* `Files/NVIDIA`
* `Corporation/NVSMI/nvidia-smi.exe`

Which is probably not what we're looking for since we're looking specifically for the nvidia-smi.exe binary